packages/legend-studio/src/stores/shared/__tests__/MockDataUtil.test.ts

```diff
@@ -20,13 +20,14 @@ import { unitTest } from '@finos/legend-shared';
 import { TEST__getTestEditorStore } from '../../EditorStoreTestUtils';
 import { flowResult } from 'mobx';
 import type { Entity } from '@finos/legend-model-storage';
+import { TEST__buildGraphWithEntities } from '@finos/legend-graph';
 
 const editorStore = TEST__getTestEditorStore();
 
 beforeAll(async () => {
   await flowResult(
-    editorStore.graphManagerState.graphManager.buildGraph(
-      editorStore.graphManagerState.graph,
+    TEST__buildGraphWithEntities(
```

Comment:
why are we doing this?

Reply:
To build `SystemModels`. Otherwise it is unable to find `temporal profile` in the graph and throwing an error